Libraries/LibWeb/Layout/Node.cpp

```diff
@@ -463,6 +465,67 @@ void NodeWithStyle::apply_style(const CSS::ComputedProperties& computed_style)
                 layer.repeat_y = repeat_value->as_background_repeat().repeat_y();
             }
 
+            if (auto blend_mode_value = value_for_layer(blend_modes, layer_index); blend_mode_value && blend_mode_value->is_keyword()) {
+                auto const num_blend_modes = blend_modes.is_value_list() ? blend_modes.as_value_list().size() : 1;
+
+                if (layer_index >= num_blend_modes) {
+                    layer.blend_mode = CSS::BlendMode::Normal;
+                } else {
+                    switch (blend_mode_value->to_keyword()) {
+                    case CSS::Keyword::Normal:
+                        layer.blend_mode = CSS::BlendMode::Normal;
+                        break;
```

Comment:
Adding an enum to Enums.json automatically generates a function that converts this for you. The whole switch can be replaced with:
```c++
layer.blend_mode = CSS::keyword_to_blend_mode(blend_mode_value->to_keyword()).value_or(CSS::BlendMode::Normal);
```
It returns Optional, so the `value_or()` is equivalent to having the `default:` branch in the switch.